warden/warden_daemon/src/client_handler/realm_client_handler.rs

```diff
@@ -203,6 +205,8 @@ mod test {
         create_example_realm_provisioning_config, MockRealmConnector, MockRealmSender,
     };
 
+    const TIMEOUT: Duration = Duration::from_secs(1);
```

Comment:
Consider parametrizing this, you already know how painfully slow is FVP...

Reply:
This one is only for testing purpouses. However there was also one hard-coded timout in the implementation, I've parametrized that one.